index.html.bak

```diff
@@ -28,11 +29,11 @@
            </div>
        </div>
   </div>
-  <script src="https://use.fontawesome.com/9690d4cb85.js"></script>
+  <script src="node_modules/font-awesome/css/font-awesome.min.css"></script>
```

Comment:
This line doesn't exist in the partials. A css file shouldn't be loaded in a script tag.